docs/docs/widgets/divider.md

```diff
@@ -50,13 +50,11 @@ Under the <b>General</b> accordion, you can set the value in the string format.
 
 ## Styles
 
-| <div style={{ width:"100px"}}> Style </div> | <div style={{ width:"100px"}}> Description </div>                                               |
-| ------------------------------------------- | ----------------------------------------------------------------------------------------------- |
-| Divider Color                               | It is used to set the color of the divider. Use hex code to set the background color.           |
-| Visibility                                  | This property is used to set the visibility of the divider. The property accepts Boolean value. |
-
-:::info
-Any property having **fx** button next to its field can be **programmatically configured**.
-:::
+| <div style="width:100px;"> Property </div>    | <div style="width:100px;"> Description </div> | <div style="width:100px;"> Configuration Options </div> |
+| ------------------------------------------- | ----------------------------------------------- | ------------------------------------------------------ |
+| Divider Color | Sets the background color of the component. | Select the color or click on fx and input code that programmatically returns a HEX color code. |
```

Comment:
Update the description to `Sets the color of the divider.`. Just update the description and not the configuration option.